gulpfile.js

```diff
@@ -1447,6 +1447,7 @@ function buildLib(defines, dir) {
     gulp.src(
       [
         "src/{core,display,shared}/*.js",
+        "src/core/xfa/*.js",
```

Comment:
I haven't tested, but wouldn't it suffice to simply do:
```suggestion
        "src/{core,display,shared}/**/*.js",
```